src/keri/core/serdering.py

```diff
@@ -304,7 +304,7 @@ class Serder:
                     Ilks.bar: Fieldage(saids={Saids.d: DigDex.Blake3_256},
                         alls=dict(v='', t='',d='', i='', dt='', r='',a=[])),
                     Ilks.exn: Fieldage(saids={Saids.d: DigDex.Blake3_256},
-                        alls=dict(v='', t='', d='', i="", p="", dt='', r='', q={},
+                        alls=dict(v='', t='', d='', i="", rp="", p="", dt='', r='', q={},
```

Comment:
What is the RP field?

Reply:
"newer KERI" this is pre 1.2.0

Rp field was added 1.2.0-dev4

Reply:
> The PR is against 1.1 - renamed to be clear.
> 
> Rp field was added 1.2.0-dev4

Sure, but what is it?

Reply:
Top level recipient field. This was discussed some time ago, hence its addition to 1.2.+
It was added to save unpacking the `a.r` field.

Reply:
> Top level recipient field. This was discussed some time ago, hence its addition to 1.2.+ It was added to save unpacking the `a.r` field.

Do you remember when?  It hasn't been added to the specs yet.

Reply:
The specs show Ver_2_0 but it looks like it’s `ri` in there, these are against Ver1__0 and Ver_1_0.